common-app-lib/src/cmr/common_app/services/kms_fetcher.clj

```diff
@@ -82,7 +86,8 @@
   (let [cache (cache/context->cache context kms-cache-key)
         gcmd-keywords-map (fetch-gcmd-keywords-map context)]
     (info "Refreshed KMS cache.")
-    (cache/set-value cache kms-cache-key gcmd-keywords-map)))
+    (cache/set-value cache kms-cache-key gcmd-keywords-map)
+    (info (format "*** the KMS cache size is %d" (cache/cache-size cache)))))
```

Comment:
two things, is this a number or a string, line 77 uses %s ; second thing, something should be different in these two lines so we can tell which is which. We may be confused as to which function wrote this?

Reply:
First off cache/cache-size returns a number, so they both should be %d.  
Secondly, I don't think we need these anymore as we now report the redis cache sizes as part of the job that reports on all of the cache sizes.  I initially used this to test why the indexer kept running out of memory. But I now know why.

Reply:
I have removed these log statements, as I think they are no longer needed.